### PR TITLE
Remove demo tool

### PR DESCRIPTION
**Pull request by seb231**
All aspects and references to demo tool now removed. Some minor updates to docs are included to match removal of demo tool and current working of the ccm.

### CODE REVIEW

File: doc/intro.md [hunk 1]
```diff
@@ -42,9 +42,6 @@ Let's define a few terms and acronyms we use throughout the library:
   - [Employment-led Cohort Component Model](#employment-led-cohort-component-model)
```
Comment (sunnytownsend) [inline doc/intro.md]: can remove housing led and employment led - not doing these

File: doc/intro.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 * Small area projections
```
Comment (sunnytownsend) [inline doc/intro.md]: remove mention of small area projections as not doing these

File: doc/intro.md [hunk 3]
```diff
@@ -75,10 +72,44 @@ where:
...
 
 ![minimal CCM](images/MVP_generic_CCM_flowchart.png)
 
+## User-defined parameters for the model
+
+Some parameters in the model can be specified by the user. The parameters, with defaults in parentheses, are:
+
+* First year of projection (2015)
+* Last year of projection (2018)
+* Start & end years of year range to average over when projecting deaths (2010-2014)
```
Comment (sunnytownsend) [inline doc/intro.md:73]: start and end years are not just for averaging over but also for trend

File: doc/intro.md [hunk 4]
```diff
@@ -75,10 +72,44 @@ where:
...
+* Start & end years of year range to average over when projecting domestic migration (2003-2014)
+* Start & end years of year range to average over when projecting international migration (2003-2014)
```
Comment (sunnytownsend) [inline doc/intro.md]: needs updating to start and end years for each in and out types of international migration

File: doc/intro.md [hunk 5]
```diff
@@ -75,10 +72,44 @@ where:
...
 ![minimal CCM](images/MVP_generic_CCM_flowchart.png)
 
+## User-defined parameters for the model
+
+Some parameters in the model can be specified by the user. The parameters, with defaults in parentheses, are:
+
+* First year of projection (2015)
+* Last year of projection (2018)
+* Start & end years of year range to average over when projecting deaths (2010-2014)
+* Start & end years of year range to average over when projecting domestic migration (2003-2014)
```
Comment (sunnytownsend) [inline doc/intro.md]: needs updating to start and end years for each in and out types of domestic migration

File: doc/intro.md (hunk 3 above)
Comment (sunnytownsend) [reply doc/intro.md:73]: need to explain somewhere the choice between average/trend - but may not belong in this PR

File: doc/intro.md [hunk 6]
```diff
@@ -75,10 +72,44 @@ where:
 
 This method involves projecting births, deaths and migrations by age and sex.
 
-See below the minimal cohort component method we aim to provide:
+See below the cohort component method we aim to provide:
 
 ![minimal CCM](images/MVP_generic_CCM_flowchart.png)
```
Comment (sunnytownsend) [inline doc/intro.md]: what is this?

Comment (sunnytownsend) [inline doc/intro.md]: possible to change the tag as says minimal?

File: doc/intro.md (hunk 3 above)
Comment (sunnytownsend) [reply doc/intro.md:73]: apologies, see this is in `intro.md`

Comment (seb231) [reply doc/intro.md:73]: Agree this should be in the next branch - made a story for it

File: doc/intro.md [hunk 7]
```diff
@@ -21,7 +19,7 @@ Let's define a few terms and acronyms we use throughout the library:
```
Comment (sunnytownsend) [inline doc/intro.md:12]: could you remove this current status bit

File: doc/intro.md [hunk 8]
```diff
@@ -117,7 +114,7 @@ See below:
 
 ![CCM - projection alternatives](images/Component_projection_alternatives.png)
 
-For estimating the value or rate for the first year of projection (jump-off year or base year), we will provide three alternatives. See below:
+For estimating the value or rate for the first year of projection (first projection year or base year), we will provide three alternatives. See below:
 
 ![CCM - base year alternatives](images/Jump_off_year_calculation_alternatives.png)
```
Comment (sunnytownsend) [inline doc/intro.md]: changed name of png - see slack

**Comment by sunnytownsend** [comment]
:shipit: (squash first?)

**Pull request merged**